images/k8s-infra/Dockerfile

```diff
@@ -0,0 +1,149 @@
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Intended to hold all dependencies needed to build, test and run all tools
+# used by wg-k8s-infra. With sufficient credentials mounted in, should be
+# capable of running those tools to test and deploy all kubernetes project
+# infrastructure managed by wg-k8s-infra.
+
+# base image
+FROM debian:buster
+
+# build args
+ARG CONFTEST_VERSION=0.25.0
+ARG GCLOUD_CHANNEL=rapid
+ARG GO_VERSION=1.16.4
+ARG JQ_VERSION=1.6
+ARG K8S_VERSION=1.20.0
```

Comment:
Any issue with 1.21.0 or 1.21.z?

Reply:
Good point, no reason it needs to be .0

I think I chose 1.20 because our clusters are still at 1.18 and I wanted to stay within 2 versions.  I'll add a comment explaining that.

Reply:
(I ended up double-checking and moved back to 1.19.x because official version skew is +/- 1 minor version)